source/extensions/filters/network/zookeeper_proxy/decoder.cc

```diff
@@ -642,22 +642,34 @@ absl::Status DecoderImpl::parseMultiRequest(Buffer::Instance& data, uint64_t& of
       break;
     }
 
-    switch (static_cast<OpCodes>(opcode.value())) {
+    const auto op = static_cast<OpCodes>(opcode.value());
```

Comment:
any motivation for one more variable? I'll prefer one less stack memory if we don't have good reason

Reply:
oh i see you need it for the macro so ignore this